web/packages/teleport/src/Welcome/NewCredentials/types.ts

```diff
@@ -38,7 +38,7 @@ export type UseTokenState = {
   redirect: () => void;
   success: boolean;
   finishedRegister: () => void;
-  privateKeyPolicyEnabled: boolean;
+  privateKeyPolicyEnabled?: boolean;
```

Comment:
why don't we just remove it (and below)? it doesn't look like it will get used anymore?